Lib/test/test_mmap.py

```diff
@@ -754,6 +764,40 @@ def test_flush_return_value(self):
             # See bpo-34754 for details.
             self.assertRaises(OSError, mm.flush, 1, len(b'python'))
 
+    def test_open_mmap(self):
```

Comment:
Why this name? We are testing `repr` not open right?

Reply:
Sorry, that's a mistake. At the very beginning I planned to make tests for both `close` cases and `open` cases, but later I found it possible to achieve exhaustive tests in only one test function.